packages/zoe/src/contractFacet/seat.js

```diff
@@ -33,7 +33,6 @@ export const makeZcfSeatAdminKit = (
         details`The seatStaging ${seatStaging} was not recognized`,
```

Comment:
I couldn't find a better place to anchor this comment. It's really about `commit`. Doing the reallocation used to be part of commit's responsibility, but it apparently no longer is. Would you mind adding a comment about what commit is taking care of, and what duties the caller is expected to follow through on?

Reply:
Yup, `commit` is only updating the currentAllocation of the seat now. Adding a comment